### DISCUSSION TITLE
Sidebar Modwheel Assign ?

### DISCUSSION THREAD

**Discussion by Qui-Gon-JooL** [SynthstromAudible/DelugeFirmware#2844] [Q&A] [answered]
Hello, I have been looking for a while how to assign a parameter of synth to the Sidebar Modwheel,
 by default it seems to me that the filter is assigned to it, I would like to remove the filter modulation from the Modwheel and assign the resonance to it, and therefore be able to modify the resonance with the Sidebar Modwheel 

I can't find how to do it... ?

**Comment by seangoodvibes** [comment, marked as answer]
> Hello, I have been looking for a while how to assign a parameter of synth to the Sidebar Modwheel, by default it seems to me that the filter is assigned to it, I would like to remove the filter modulation from the Modwheel and assign the resonance to it, and therefore be able to modify the resonance with the Sidebar Modwheel
> 
> I can't find how to do it... ?

It depends on each synth what MPE Y is mapped to. But you can change it via the Mod Matrix Menu and using the Modulation Source menu

1. Remove what MPE Y is currently patched to:

a. Press Settings Encoder
b. Scroll down to the Mod Matrix menu
c. Select the MPE Y -> Parameter that you want to remove
d. Change the value to 0 

2. Add new MPE Y mapping

a. Shift + Parameter / Enter Parameter Menu
b. Press Select while in the Parameter Menu
c. Select MPE Y in the Modulate With menu (modulation source)
d. Set a value for MPE Y -> Parameter

**Comment by Qui-Gon-JooL** [comment]
Yes that's right, thank you.

Is there any way to assign the Y modulation to a knob?

**Reply by seangoodvibes** [reply]
> Yes that's right, thank you.
> 
> Is there any way to assign the Y modulation to a knob?

You just hold learn and turn a gold knob while you're in the MPE Y -> Parameter menu.

**Reply by Qui-Gon-JooL** [reply]
I expressed myself badly, I would like to be able to assign the Mod wheel to a knob rather than to the sidebar, would that be possible?

Because the sidebar works in jumps. While the knob is smooth.

**Reply by seangoodvibes** [reply]
> I expressed myself badly, I would like to be able to assign the Mod wheel to a knob rather than to the sidebar, would that be possible?
> 
> Because the sidebar works in jumps. While the knob is smooth.

Yes that's what I explained. You open the sound menu, go to the mod matrix, find the MPE Y -> parameter patch cable, press select to enter the patch cable editing menu, hold learn and turn a gold knob to learn that patch cable to the gold knob.

**Reply by Qui-Gon-JooL** [reply]
If I do what you say, I don't put the mod wheel on the knob, but I put on the knob the amount of modulation of the parameter linked to the mod wheel, and of this parameter alone. 

What I would like to do is be able to do like the blue sidebar (modwheel) but on a knob.

example, I connected the pitch of one of the osc, the filter and the resonance to the Y, when I move the blue sidebar (mod wheel), I modulate these three parameters at the same time by the quantity defined independently (link to Y), It's perfect.

But I would like to do the same thing but move a knob and not the sidebar to avoid jumps.

This may not be possible...

**Reply by seangoodvibes** [reply]
> If I do what you say, I don't put the mod wheel on the knob, but I put on the knob the amount of modulation of the parameter linked to the mod wheel, and of this parameter alone.
> 
> What I would like to do is be able to do like the blue sidebar (modwheel) but on a knob.
> 
> example, I connected the pitch of one of the osc, the filter and the resonance to the Y, when I move the blue sidebar (mod wheel), I modulate these three parameters at the same time by the quantity defined independently (link to Y), It's perfect.
> 
> But I would like to do the same thing but move a knob and not the sidebar to avoid jumps.
> 
> This may not be possible...

ah yea not possible

**Reply by Qui-Gon-JooL** [reply]
Thanks anyway
